libs/utils/utils.js

```diff
@@ -1059,20 +1105,25 @@ async function checkForPageMods() {
   const { init } = await import('../features/personalization/personalization.js');
   await init({
     mepParam, mepHighlight, mepButton, pzn, promo, target,
-  });
+  }, targetInteractionPromise);
 }
 
 async function loadPostLCP(config) {
+  const enablePersV2 = enablePersonalizationV2();
+  if (enablePersV2 && delayedMartech) {
+    await loadMartech();
```

Comment:
It seems personalization also loads martech (pre-LCP) are both really required or can we be smarter about when we load that?

Reply:
Addressed